src/routes/signup/+page.server.ts

```diff
@@ -5,8 +5,10 @@ import {PUBLIC_API_URL} from "$env/static/public";
 
 const numSaltRound = 8;
 
-export const load: PageServerLoad = async () => {
-	// nothing needs to happen here
+export const load: PageServerLoad = async ({ locals }) => {
+	if (locals.user.isLoggedIn) {
```

Comment:
Is this a variable on  the client side or frontend server side?

Reply:
See above